edge/pkg/edged/edged.go

```diff
@@ -581,8 +581,8 @@ func newEdged(enable bool) (*edged, error) {
 		ed.cadvisor,
 		cm.NodeConfig{
 			CgroupDriver:                 edgedconfig.Config.CGroupDriver,
-			SystemCgroupsName:            edgedconfig.Config.CGroupDriver,
-			KubeletCgroupsName:           edgedconfig.Config.CGroupDriver,
+			SystemCgroupsName:            "",
+			KubeletCgroupsName:           "",
```

Comment:
better to be defined as config options?

Reply:
These two fields now have no clear purpose on the edge side, let's expose it if needed later.

Reply:
Finally, exposed them ;-)